packages/@n8n/backend-common/src/modules/module-registry.ts

```diff
@@ -113,6 +113,22 @@ export class ModuleRegistry {
 		}
 	}
 
+	async shutdownModule(moduleName: ModuleName) {
```

Comment:
This function is for future right? I.e. we are not calling the shutdown twice now? Who is responsible for calling this?